app/celery/process_pinpoint_inbound_sms.py

```diff
@@ -31,10 +30,6 @@ def process_pinpoint_inbound_sms(
 ):
     provider_name = 'pinpoint'
 
-    if not is_feature_enabled(FeatureFlag.PINPOINT_INBOUND_SMS_ENABLED):
-        current_app.logger.info('Pinpoint inbound SMS toggle is disabled, skipping task')
-        return True
```

Comment:
This appears to be off in perf (flag not present). I think it's still okay to remove, just noting the change here.

Reply:
I concur.

Reply:
@keisterj-oddball Says he's okay with the change.